src/OpcPlcServer.cs

```diff
@@ -91,15 +95,24 @@
         Logger.LogInformation("Log file: {logFileName}", Path.GetFullPath(Config.LogFileName));
         Logger.LogInformation("Log level: {logLevel}", Config.LogLevelCli);
 
-        // Show version.
+        // Show OPC PLC version.
```

Comment:
>Show [](http://example.com/codeflow?start=11&length=4)

Pls check why this changes are showing up, main already contains them. Maybe a fwd integration mistake?

Reply:
fixed merge with main